Sources/Formatters/FormatPatternValidator.swift

```diff
@@ -27,24 +28,39 @@ class FormatPatternValidator {
      
      - Returns: `Result` with `Void` and the error message.
      */
-    func validate(for pattern: String) -> Result<Void, String> {
-        if pattern.contains("%%") {
-            return .error(buildErrorMessage(characters: "%%"))
+    func validate(for pattern: String) -> ValidationResult {
+        if let match = illegalCharactersList.filter({ pattern.contains($0) }).first {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Scior/Linna/blob/afc19061de2e5644dd827f6934b6a89f71db98b7/Sources/Formatters/FormatPatternValidator.swift#L32"></span>Prefer using <code>.first(where:)</code> over <code>.filter { }.first</code> in collections.</td>
    </tr>
  </tbody>
</table>